src/app/features/tasks/short-syntax.ts

```diff
@@ -18,9 +18,13 @@ type DueChanges = {
   dueWithTime?: number;
 };
 
-const SHORT_SYNTAX_TIME_REG_EX =
-  /(?:\s|^)t?((\d+(?:\.\d+)?[mhd])(?:\s*\/\s*(\d+(?:\.\d+)?[mhd]))?(?=\s|$))/i;
-// NOTE: should come after the time reg ex is executed so we don't have to deal with those strings too
+const CH_TSP = '/';
```

Comment:
Thought I'd separate the special character used to delimit time spent and estimated for easier identification going forward.